src/client/Client.js

```diff
@@ -135,7 +135,8 @@ class Client extends BaseClient {
     Object.defineProperty(this, 'token', { writable: true });
     if (!browser && !this.token && 'DISCORD_TOKEN' in process.env) {
       /**
-       * Authorization token for the logged in bot
+       * Authorization token for the logged in bot.
+       * Gets set to `process.env.DISCORD_TOKEN` when instantiating the client
```

Comment:
```suggestion
       * If present, this defaults to `process.env.DISCORD_TOKEN` when instantiating the client
```
This sounds better in my opinion but I'd like to get feedback before committing @vladfrangu @kyranet

Reply:
I like this alternative 👍